src/workloads/query/CollectionLevelDiagnosticCommands.yml

```diff
@@ -0,0 +1,176 @@
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: *CollectionCount
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          int: &integer {^Choose: {from: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], deterministic: true}}
+          string: &string {^Choose: {from: ["swimswimswimswimswim", "sinksinksinksink"], deterministic: true}}
+          obj: {int: *integer, string: *string}
+
+# Phase 2: Ensure all data is synced to disk.
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+- Name: DiagnosticTopCommand
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3, 5, 7, 9]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 100
+        Thread: 1
+        Database: admin
+        Operations:
+        - OperationMetricsName: DiagnosticTopCommand
+          OperationName: AdminCommand
+          OperationCommand:
+            top: 1
+
+- Name: AggForDiagnosticSetup
+  Type: CrudActor
+  Database: *Database
+  CollectionCount: *CollectionCount
+  Threads: *CollectionCount
+  Phases:
+    OnlyActiveInPhases:
+      Active: [4]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        CollectionCount: *CollectionCount
+        Operations:
+        - OperationMetricsName: AggForDiagnosticSetup
+          OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$group: {_id: "$int", sumOfDoubles: {$sum: "$double"}}}, {$project: {string: 1}}, {$match: { string : "swimswimswimswimswim" }}]
+
+- Name: QueryForDiagnosticSetup
+  Type: CrudActor
+  Database: *Database
+  CollectionCount: *CollectionCount
+  Threads: *CollectionCount
+  Phases:
+    OnlyActiveInPhases:
+      Active: [6]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 10
+        Database: *Database
+        CollectionCount: *CollectionCount
+        Operation:
+          OperationName: find
+          OperationCommand:
+            Filter: {string: "sinksinksinksink"}
+            Options:
+              Limit: 100
+
+- Name: UpdateManyForDiagnosticSetup
+  Type: CrudActor
+  Database: *Database
+  CollectionCount: *CollectionCount
+  Threads: *CollectionCount
+  Phases:
+    OnlyActiveInPhases:
+      Active: [8]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 5
+        Database: *Database
+        CollectionCount: *CollectionCount
+        Operation:
+          OperationName: updateMany
+          OperationCommand:
+            Filter: { obj.int: {$lt: 5} }
+            Update: {$inc: {x: 1}}
+
+- Name: DeleteManyForDiagnosticSetup
+  Type: CrudActor
+  Database: *Database
+  CollectionCount: *CollectionCount
+  Threads: *CollectionCount
+  Phases:
+    OnlyActiveInPhases:
+      Active: [8]
```

Comment:
This is same phase as the above one, do you want to update and delete concurrently?

Reply:
That was an oversight, thanks for catching this. Done.